app/controllers/admin/inbound_api_configurations_controller.rb

```diff
@@ -0,0 +1,46 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Admin
+  class InboundApiConfigurationsController < ::ApplicationController
+    def index
+      @confs = HmisExternalApis::InboundApiConfiguration.well_ordered
+      @pagy, @confs = pagy(@confs, items: 20)
+    end
+
+    def new
+      @conf = HmisExternalApis::InboundApiConfiguration.new
+    end
+
+    def create
+      @conf = HmisExternalApis::InboundApiConfiguration.new(permitted_params)
+
+      if @conf.save
+        flash[:alert] = "Your key is #{@conf.plain_text_api_key}. It will not be shown again"
```

Comment:
I wonder if we should somehow make this more obvious on the page.  I feel like people just ignore the alert message at the top.  Could we somehow surface it in the main body of the page without storing it?  Maybe we store it in redis with a 30 second expiry or something?

Reply:
I was thinking it wasn't obvious enough either, but I wanted to hear what people think. I like the redis idea.

Reply:
Eh I think its ok.. if they missed it they can always re create the key. But it wouldn't hurt to make it more prominent, if its not a big lift. Redis seems like a good idea. Another option if possible could be to just move the flash message down so its right above the table.